Cython/Build/IpythonMagic.py

```diff
@@ -179,8 +179,11 @@ def f(x):
 
     @magic_arguments.magic_arguments()
     @magic_arguments.argument(
-        '-a', '--annotate', nargs='?', const=1, type=int,
-        help="Produce a colorized HTML version of the source. Use --annotate=2 to include complete generated C/C++-code."
+        '-a', '--annotate', nargs='?', const="default", type=str,
+        choices={"default","show_entire_c_code"},
+        help="Produce a colorized HTML version of the source. "
+             "Use --annotate=show_entire_c_code to include entire "
+             "generated C/C++-code."
```

Comment:
@scoder are you Ok with this solution and/or maybe do you have better names?

Reply:
Let's keep the variable names as they are now and just change this to `--annotate=fullc` to keep it typeable.